src/deepsparse/benchmark/benchmark_model.py

```diff
@@ -392,13 +394,42 @@ def benchmark_model(
         "benchmark_result": benchmark_result,
     }
 
+    parsed_export_dict = _parse_export_dict_engine_key(export_dict)
+
     # Export results
     if export_path:
         _LOGGER.info("Saving benchmark results to JSON file at {}".format(export_path))
         with open(export_path, "w") as out:
-            json.dump(export_dict, out, indent=2)
-
-    return export_dict
+            json.dump(parsed_export_dict, out, indent=2)
+
+    return parsed_export_dict
+
+
+def _parse_export_dict_engine_key(payload):
```

Comment:
@kazukageorge it would be good to migrate this over to a regex implementation to make it more obvious what the expected input and subsequent output are

Reply:
OOO maybe you can get away with `yaml.load`... that would help to make it explicit what we expect to find in the engine key and make it more flexible in the event that the engine key is not normal (e.g. property order varies)